paybybot2/bot.py

```diff
@@ -18,11 +16,11 @@ class Bot:
 
     def __init__(self, username, password):
         r = requests.post(
-            "https://api.paybyphone.com/identity/token",
+            "https://auth.paybyphoneapis.com/token",
             headers={
                 **self.base_headers,
                 "Accept": "application/json, text/plain, */*",
-                "Authorization": "null",
+                "X-Pbp-ClientType": "WebApp",
```

Comment:
Keeping ``Authorization`` header as ``null`` string now makes request to fail. Also ``X-Pbp-ClientType`` must now be provided.